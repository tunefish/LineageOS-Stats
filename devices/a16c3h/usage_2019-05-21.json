{
  "1.0-20180727-unofficial-a16c3h": 1,
  "1.0-20180727-unofficial-andromax-a16c3h": 2,
  "12.1-20160829-reborn-el-maleek-a16c3h": 1,
  "12.1-20161113-unofficial-konstakang-a16c3h": 1,
  "13.0-20170119-unofficial-a16c3h": 7,
  "13.0-20170207-unofficial-a16c3h": 1,
  "13.0-20170207-unofficial-konstakang-a16c3h": 3,
  "13.0-20170207-unofficial-konstakang-y560": 1210,
  "13.0-20170227-unofficial-a16c3h": 10,
  "13.0-20170902-official-jalebi": 1,
  "13.0-20170902-unofficial-jalebi": 178,
  "13.0-20171028-nightly-a16c3h": 12,
  "13.0-unofficial-google\u2122": 1,
  "14.0-20170207-unofficial-konstakang-y560": 1,
  "14.1-20170304-unofficial": 1,
  "14.1-20180108-unofficial-a16c3h": 9,
  "14.1-20180221-a16c3h-extended": 8,
  "14.1-20180304-microg-a16c3h": 1,
  "14.1-20180425-unofficial-a16c3h": 1,
  "14.1-20180907-microg-a16c3h": 17,
  "14.1-20190105-nightly-a16c3h": 90,
  "14.1-20190205-nightly-a16c3h": 2,
  "a16c3h-userdebug 7.1.2 njh47f 16f36b7273 release-keys": 14,
  "a16c3h-userdebug 7.1.2 njh47f 91a96bc9b4 release-keys": 26,
  "lineage-20170313-unofficial-a16c3h": 1,
  "nuclearom-v-unofficial[7.1.2]--20171117-2103": 2
}