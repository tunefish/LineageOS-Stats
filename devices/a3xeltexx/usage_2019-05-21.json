{
  "14.1-20170416-unofficial-a3xeltexx": 7,
  "14.1-20170703-unofficial-a3xeltexx": 4,
  "14.1-20170706-unofficial-a3xeltexx": 1,
  "14.1-20170709-unofficial-a3xeltexx": 1,
  "14.1-20170711-unofficial-a3xeltexx": 16,
  "14.1-20170713-unofficial-a3xeltexx": 1,
  "14.1-20170729-unofficial-a3xeltexx": 1,
  "14.1-20170807-unofficial-a3xeltexx": 1,
  "14.1-20170826-unofficial-a3xeltexx": 5,
  "14.1-20170907-unofficial-a3xeltexx": 51,
  "14.1-20170909-unofficial-a3xeltexx": 3,
  "14.1-20171025_105805-unofficial-a3xeltexx": 3,
  "14.1-20171027_160221-unofficial-a3xeltexx": 1,
  "14.1-20171101_122209-unofficial-a3xeltexx": 3,
  "14.1-20171114_052935-unofficial-a3xeltexx": 2,
  "14.1-20171123_144604-unofficial-a3xeltexx": 1,
  "14.1-20171201_185323-unofficial-a3xeltexx": 29,
  "14.1-20180411-unofficial-a3xeltexx": 249,
  "14.1-20180726-unofficial-a3xeltexx": 1,
  "15.1-20180228-unofficial-a3xeltexx": 13,
  "15.1-20180302-unofficial-a3xeltexx": 3,
  "15.1-20180305-unofficial-a3xeltexx": 7,
  "15.1-20180320-unofficial-a3xeltexx": 3,
  "15.1-20180329-unofficial-a3xeltexx": 32,
  "15.1-20180410-unofficial-a3xeltexx": 315,
  "15.1-20180413-unofficial-a3xeltexx": 489,
  "15.1-20180606-unofficial-a3xeltexx": 12,
  "15.1-20180625-unofficial-a3xeltexx": 1,
  "15.1-20180705-unofficial-a3xeltexx": 1,
  "15.1-20180711-unofficial-a3xeltexx": 1,
  "15.1-20180713-unofficial-a3xeltexx": 3,
  "15.1-20180729-unofficial-a3xeltexx": 7,
  "15.1-20180803-unofficial-a3xeltexx": 13,
  "15.1-20180822-unofficial-a3xeltexx": 2,
  "15.1-20180826-unofficial-a3xeltexx": 4,
  "15.1-20180830-unofficial-a3xeltexx": 9,
  "15.1-20180908-unofficial-a3xeltexx": 9,
  "15.1-20180914-unofficial-a3xeltexx": 26,
  "15.1-20180921-unofficial-a3xeltexx": 59,
  "15.1-20180928-unofficial-a3xeltexx": 21,
  "15.1-20181014-unofficial-a3xeltexx": 32,
  "15.1-20181109-unofficial-a3xeltexx": 47,
  "15.1-20181127-unofficial-a3xeltexx": 348,
  "15.1-20190112-unofficial-a3xeltexx": 190,
  "15.1-20190113-dnjl-a3xeltexx": 1,
  "15.1-20190120-dnjl-a3xeltexx": 1
}