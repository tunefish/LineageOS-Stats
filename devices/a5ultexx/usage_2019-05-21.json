{
  "14.1-20170114-unofficial-a5ultexx": 22,
  "14.1-20170216-unofficial-a5ultexx": 181,
  "14.1-20170817-unofficial-a5ultexx": 173,
  "14.1-20170822-unofficial-a5ultexx": 9,
  "14.1-20170906-unofficial-a5ultexx": 176,
  "14.1-20171231_020714-unofficial-a5ultexx": 93,
  "14.1-20180820-unofficial-a5ultexx": 74,
  "14.1-20180825-unofficial-a5ultexx": 8,
  "14.1-20180826-unofficial-a5ultexx": 7,
  "14.1-20180829-unofficial-a5ultexx": 10,
  "14.1-20180830-unofficial-a5ultexx": 10,
  "14.1-20181130-unofficial-a5ultexx": 1,
  "14.1-20181221-unofficial-a5ultexx": 1,
  "14.1-20181228-unofficial-a5ultexx": 1,
  "15.1-20180210-unofficial-a5ultexx": 299,
  "16.0-20181015-unofficial-a5ultexx": 1,
  "16.0-20181017-unofficial-a5ultexx": 10,
  "16.0-20181102-unofficial-a5ultexx": 142,
  "16.0-20181201-unofficial-a5ultexx": 1,
  "16.0-20190129-unofficial-a5ultexx": 97,
  "16.0-20190220-unofficial-a5ultexx": 25,
  "16.0-20190222-unofficial-a5ultexx": 4,
  "16.0-20190223-unofficial-a5ultexx": 1,
  "16.0-20190225-unofficial-a5ultexx": 60
}