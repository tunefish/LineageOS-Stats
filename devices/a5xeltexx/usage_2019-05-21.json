{
  "14.1-20170825-unofficial-a5xeltexx": 1,
  "14.1-20170827-unofficial-a5xeltexx": 70,
  "15.1-20180209-unofficial-a5xeltexx": 58,
  "15.1-20180221-unofficial-a5xeltexx": 7,
  "15.1-20180302-unofficial-a5xeltexx": 2,
  "15.1-20180304-unofficial-a5xeltexx": 5,
  "15.1-20180305-unofficial-a5xeltexx": 3,
  "15.1-20180320-unofficial-a5xeltexx": 14,
  "15.1-20180328-unofficial-a5xeltexx": 33,
  "15.1-20180408-unofficial-a5xeltexx": 1005,
  "15.1-20181218-unofficial-a5xeltexx": 5,
  "15.1-20181223-unofficial-a5xeltexx": 4,
  "15.1-20190110-unofficial-a5xeltexx": 3,
  "15.1-20190111-unofficial-a5xeltexx": 1
}