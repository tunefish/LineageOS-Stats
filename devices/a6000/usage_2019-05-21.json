{
  "14.1-20170125-unofficial-a6000": 27,
  "14.1-20170205-unofficial-a6000": 35,
  "14.1-20170210-unofficial-a6000": 27,
  "14.1-20170225-unofficial-a6000": 198,
  "14.1-20170228-unofficial-a6000": 6,
  "14.1-20170318-unofficial-a6000": 1,
  "14.1-20170403-unofficial-a6000": 48,
  "14.1-20170422-nightly-a6000": 22,
  "14.1-20170514-unofficial-a6000": 14,
  "14.1-20170607-unofficial-a6000": 6,
  "14.1-20170614-unofficial-a6000": 8,
  "14.1-20170706-unofficial-a6000": 63,
  "14.1-20170706-unofficial-dev_harsh1998-a6000": 7,
  "14.1-20170713-unofficial-a6000": 2,
  "14.1-20170721-unofficial-a6000": 2,
  "14.1-20170731-unofficial-hyperteam-a6000": 7,
  "14.1-20170813-unofficial-a6000": 15,
  "14.1-20170831-unofficial-a6000": 263,
  "14.1-20171012-unofficial-a6000": 1,
  "14.1-20180108-official-a6000": 1,
  "14.1-20180108-unofficial-a6000": 50,
  "14.1-20181107-nightly-athene": 1,
  "15.1-20180210-hyper-a6000": 29,
  "15.1-20180216-hyper-a6000": 145,
  "15.1-20180219-hyper-a6000": 24,
  "15.1-20180308-hyper-a6000": 48,
  "15.1-20180413-hyper-a6000": 118,
  "15.1-20180514-unofficial-a6000": 1,
  "15.1-20180528-hyper-a6000": 1,
  "15.1-20180604-hyper-a6000": 3,
  "15.1-20180611-hyper-a6000": 22,
  "15.1-20180720-hyper-a6000": 11,
  "15.1-20180720-unofficial-a6000": 11,
  "15.1-20180724-hyper-a6000": 92,
  "15.1-20180807-hyper-stable-a6000": 795,
  "15.1-20190220-unofficial-a6000": 1,
  "16.0-20190301-unofficial-a6000": 20,
  "16.0-20190302-unofficial-a6000": 85,
  "16.0-20190305-unofficial-a6000": 31,
  "nuclearom-unofficial[7.1.2]--20181211": 2,
  "nuclearom-v-unofficial[7.1.2]--20171117-2103": 47
}