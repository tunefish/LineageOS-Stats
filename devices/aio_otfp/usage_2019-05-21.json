{
  "13.0-20170114-unofficial-aio_otfp": 3,
  "14.1-20170117-unofficial-aio_otfp": 3,
  "14.1-20170214-unofficial-aio_otfp": 169,
  "14.1-20170312-unofficial-aio_otfp": 24,
  "14.1-20170408-unofficial-aio_otfp": 18,
  "14.1-20170420-unofficial-aio_otfp": 1,
  "14.1-20170430-unofficial-aio_otfp": 16,
  "14.1-20170526-unofficial-aio_otfp": 1,
  "14.1-20170602-unofficial-aio_otfp": 2,
  "14.1-20170616-unofficial-aio_otfp": 22,
  "14.1-20170617-unofficial-aio_otfp": 15,
  "14.1-20170702-unofficial-aio_otfp": 13,
  "14.1-20170729-unofficial-aio_otfp": 2,
  "14.1-20170820-unofficial-aio_otfp": 9,
  "14.1-20171001-unofficial-aio_otfp": 2,
  "14.1-20171031_105651-unofficial-aio_otfp": 8,
  "14.1-20171203_053532-unofficial-aio_otfp": 25,
  "14.1-20171216_141341-unofficial-aio_otfp": 79,
  "14.1-20180127-official-aio_otfp": 52,
  "14.1-20180127-unofficial-aio_otfp": 546,
  "14.1-20190114-unofficial-aio_otfp": 91,
  "14.1-20190305-unofficial-aio_otfp": 1,
  "15.1-20180221-official-aio_otfp": 1,
  "15.1-20180221-unofficial-aio_otfp": 725,
  "15.1-20180309-unofficial-aio_otfp": 37,
  "15.1-20180413-unofficial-aio_otfp": 1,
  "15.1-20180421-unofficial-aio_otfp": 19,
  "15.1-20180509-unofficial-a7000": 26,
  "15.1-20180509-unofficial-aio_otfp": 74,
  "15.1-20180509-unofficial-aio_row": 24,
  "15.1-20180509-unofficial-p6000": 39,
  "15.1-20180511-unofficial-aio_otfp": 70,
  "15.1-20180627-unofficial-aio_otfp": 5,
  "15.1-20180713-unofficial-aio_otfp": 280,
  "7.0-20190124-unofficial-aio_otfp": 55,
  "lineageos-15.1-by-derak1129": 1,
  "lineageos-15.1-by-tanish2k09": 1
}