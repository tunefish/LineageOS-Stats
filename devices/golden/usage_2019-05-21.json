{
  "12.1-20160524-unofficial-golden": 1,
  "13.0-20170118-unofficial-golden": 5,
  "13.0-20170121-unofficial-golden": 2,
  "13.0-20170129-unofficial-golden": 129,
  "13.0-20170224-unofficial-golden": 10,
  "13.0-20170302-unofficial-golden": 20,
  "13.0-20170310-unofficial-golden": 95,
  "13.0-20170328-unofficial-golden": 972,
  "13.0-20170420-unofficial-golden": 25,
  "13.0-20180315-unofficial-golden": 3,
  "13.0-20180620-unofficial-golden": 1,
  "14.1-20170122-unofficial-golden": 217,
  "14.1-20170318-unofficial-golden": 8,
  "14.1-20170402-unofficial-golden": 22,
  "14.1-20170415-unofficial-golden": 8,
  "14.1-20170419-unofficial-golden": 13,
  "14.1-20170503-unofficial-golden": 4,
  "14.1-20170507-unofficial-golden": 48,
  "14.1-20170516-unofficial-golden": 2,
  "14.1-20170522-unofficial-golden": 10,
  "14.1-20170609-unofficial-golden": 890,
  "14.1-20170610-unofficial-golden": 21,
  "14.1-20170628-unofficial-golden": 899,
  "14.1-20170724-unofficial-golden": 68,
  "14.1-20170817-unofficial-golden": 82,
  "14.1-20170920-unofficial-golden": 27,
  "14.1-20171011-unofficial-golden": 210,
  "14.1-20171215_192311-official-golden": 90,
  "14.1-20171215_192311-unofficial-golden": 1675,
  "14.1-20180109_002536-unofficial-golden": 394,
  "14.1-20181014-unofficial-golden": 14,
  "15.1-20181030-unofficial-golden": 11,
  "android one(lineageos14.1)": 1
}